NetKAN/TweakScaleRescaled.netkan

```diff
@@ -0,0 +1,32 @@
+spec_version: v1.18
+identifier: TweakScaleRescaled
+name: TweakScale Rescaled
+abstract: >-
+  Rescale parts in the vessel editor.  More features, fewer bugs.
+$kref: '#/ckan/spacedock/3514'
+$vref: '#/ckan/ksp-avc/GameData/TweakScale/TweakScale.version'
+release_status: testing
+license: CC-BY-NC-SA
+author:
+  - Biotronic
+  - pellinor
+  - JonnyOThan
+tags:
+  - plugin
+  - convenience
+depends:
+  - name: Harmony2
+    min_version: 2.2
+  - name: ModuleManager
+  - name: TweakScaleRescaled-Redist
+recommends:
+  - name: TweakScaleRescaled-SafetyNet
+  - name: KSPCommunityFixes
+conflicts:
+  - name: TweakScale
+  - name: TweakScaleCompanion
```

Comment:
Glancing at the other fork, should this also conflict with `KSP-Recall`?
What about `TweakscaleMakingHistoryConfigs`?

Reply:
I don't think it *actually* conflicts with KSP-Recall, though there's not much of a reason to have it as long as you have KSPCommunityFixes.  I'll have to check the MH configs.